### ISSUE TITLE
.NET Framework 4.5+ not supported

### ISSUE THREAD

**Issue by FenceDad** [ShipEngine/shipengine-dotnet#57]
With the new SDK, is the plan going forward to not support .NET Framework 4, 4.5+ and to only support .NET Standard?  This is a big problem for us if that's the case.

**Comment by sushithegreat** [comment]
Yes, we only plan to support .NET 5.0 and above since the SDK has moved to `async` methods. Since you're running an earlier version of .NET you can make calls directly to the ShipEngine API instead, I'm happy to help out with questions you may have on that

**Comment by FenceDad** [comment]
.NET 5 is a SIGNIFICANT upgrade for most large systems.  ASYNC methods have been available to .NET 4 and 4.5 for some time.  Making calls directly to the API can be difficult as that is one more piece of code we maintain. We use and pay for ShipEngine because of the SDK support and integration.

**Comment by AnneOReilly** [comment]
Hey @FenceDad!  We'll be reaching out to setup a call to talk through your questions.

**Comment by sushithegreat** [comment]
My initial response to this issue had incorrect information. .NET Framework 4.6.1 is the first to support .NET Standard 2.0, so we can only support .NET Framework 4.6.1 and above. 

If you want to read more about this, I recommend [this article](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0#net-5-and-net-standard). The tl;dr is that Standard allows us to support a wider range of features for more versions of Framework.
